spec.bs

```diff
@@ -868,12 +902,16 @@ A <dfn export for=fencedframetype>destination event</dfn> is either a
     : [=request/credentials mode=]
     :: `"omit"`
 
-  1. [=Fetch=] |request|.
+    : [=request/Attribution Reporting eligibility=]
+    :: |attributionReportingEligibility|
+
+  1. [=Fetch=] |request| with [=fetch/processResponse=] being |processResponse| if it is not null
+     and [=fetch/useParallelQueue=] being |useParallelQueue|.
 
     Note: This algorithm can be invoked from while [=in parallel=] or while on a {{Document}}'s main
-    thread. To handle the [=in parallel=] invocation correctly, ordinarily we would invoke [=fetch=]
-    with [=fetch/useParallelQueue=] set to true, but since we don't pass in any of the <span
-    class=allow-2119>optional</span> [=response=]-handling algorithms, there is no need to do this.
+    thread. To handle the [=in parallel=] invocation correctly, we invoke [=fetch=]
+    with [=fetch/useParallelQueue=] set to true when the <span class=allow-2119>optional</span>
+    [=response=]-handling algorithm is used, otherwise there is no need to do this.
```

Comment:
nit: `otherwise there is no need to do this even when this algorithm is running in parallel in other instances.`